xml/admin_operating_services.xml

```diff
@@ -10,245 +10,204 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
-   <dm:release>SES 6</dm:release>
+   <dm:release>SES 7</dm:release>
   </dm:docmanager>
  </info>
  <para>
-  You can operate &ceph; services either using &systemd; or using &deepsea;.
+  You can operate &ceph; services on a daemon, node, or cluster level.
+  Depending on which approach you need, use &cephadm; or the
+  <command>systemctl</command> command.
  </para>
- <sect1 xml:id="operate-services-systemd">
-  <title>Operating &ceph; Cluster Related Services Using &systemd;</title>
+ <sect1 xml:id="cha-ceph-operating-individual">
+  <title>Operating Individual Services</title>
 
   <para>
-   Use the <command>systemctl</command> command to operate all &ceph; related
-   services. The operation takes place on the node you are currently logged in
-   to. You need to have &rootuser; privileges to be able to operate on &ceph;
-   services.
+   If you need to operate an individual service, identify it first:
   </para>
 
-  <sect2 xml:id="ceph-operating-services-targets">
-   <title>Starting, Stopping, and Restarting Services Using Targets</title>
-   <para>
-    To simplify starting, stopping, and restarting all the services of a
-    particular type (for example all &ceph; services, or all MONs, or all OSDs)
-    on a node, &ceph; provides the following &systemd; unit files:
-   </para>
-<screen>&prompt.cephuser;ls /usr/lib/systemd/system/ceph*.target
-ceph.target
-ceph-osd.target
-ceph-mon.target
-ceph-mgr.target
-ceph-mds.target
-ceph-radosgw.target
-ceph-rbd-mirror.target</screen>
-   <para>
-    To start/stop/restart all &ceph; services on the node, run:
-   </para>
 <screen>
-&prompt.root;systemctl start ceph.target
-&prompt.root;systemctl stop ceph.target
-&prompt.root;systemctl restart ceph.target
+&prompt.cephuser;ceph orch ps
```

Comment:
Perhaps add a small note that this (and most other) commands support various different formats `[--format json|json-pretty|yaml]`

Reply:
thanks for heads-up